contrib/hadoop-store-builder/src/java/voldemort/store/readonly/disk/KeyValueWriter.java

```diff
@@ -25,6 +25,13 @@
 // Interface used by reducers to layout the datqa on disk
 public interface KeyValueWriter<K, V> {
 
+    public static final String COMPRESSION_CODEC = "GZIP";
```

Comment:
It might be cleaner to put all these compression-related constants in a parameterized enum, though I don't think that's a blocker for merging. Let's just maybe add a TODO so we keep it in mind if/when we want to add support for other compression types?